### ISSUE TITLE
Create JSON design token output

### ISSUE THREAD

**Issue by zttodd** [indiana-university/rivet-source#317]

**Comment by levimcg** [comment]
Wanted to jump in here and add a bit more context to this issue. Ideally, I'd like to see the tokens we output (ship with the npm package) as JSON have the shape of the data transformed into something that is a little easier to use in a real-world setting.

For example, we'll be [using 11ty](https://www.11ty.dev/) for the next version of the documentation site. I'd like to easily be able to drop the `tokens.json` out put file into the [_data folder](https://www.11ty.dev/docs/data-global/) in my 11ty project then loop through values in arrays of objects that contain the token names and hex code values. We did something very similar (although a more lo-fi version) here in [the current Rivet docs site](https://github.com/indiana-university/rivet-docs/blob/develop/layouts/shortcodes/sassdocs.html#L90). I'd like to be able to replicate a set up like this, but with the actual generated tokens output (.json file).

Right now when you use style dictionary to output a JSON version of the tokens it just copies the shape of the source tokens 1:1. Here's an example:

```json
{
  "color": {
    "black": {
      "base": {
        "value": "{color.black.800.value}"
      },
      "000": {
        "value": "#f7f7f8"
      },
      "100": {
        "value": "#ebecee"
      }
  }
}
```

This structure makes it hard to loop/use array methods on the data. If you take the 11ty example I mention above, you wouldn't be able to easily loop over an array of color values in your templates because the color values are not structured in arrays. You'd end up needing to first do some `Object.keys(tokens)` stuff to try and iterate over the data and that would be no fun.

Ideally, in the .json output file values, might be structured something like this:

```json
{
  "colors": [
    {
      "name": "color-black-000",
      "value": "#f7f7f8"
    },
    {
      "name": "color-black-100",
      "value": "#ebecee"
    },
    {
      "name": "color-black-200",
      "value": "#c4c7cc"
    }
  ],
  "ts": [
    {
      "name": "12",
      "value": ".75rem"
    },
    {
      "name": "14",
      "value": ".875rem"
    }
  ]
}
```

With a data structure like this in place, it would be much easier to iterate over groups of tokens. For example, with this .json file in place in my `_data` folder I could then loop over them in my nunjucks, liquid, etc templates like:

```liquid
<ul>
  {% for color in colors %}
  <li>
    Name: {{ color.name }}
    Value: {{ color.value }}
  </li>
  {% endfor %}
</ul>
```

or in JavaScript

```js
// In a Nodejs environment (think build tools, etc.)
const tokens = require('path/to/tokens.json');

tokens.colors.forEach(color => doStuffWithColor(color));
```

Happy to chat through this about fine-tuning the shape of the output to whatever makes the most sense. I'm sure there will be a few cases where the recommendations above don't make sense.

**Comment by illusivesunrae** [comment]
Released with PR #355.
